packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx

```diff
@@ -50,4 +51,11 @@ export const Default: StoryObj<typeof Checkbox> = {
   args: {
     label: "I agree to the terms and conditions",
   },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const el = await canvas.findByLabelText(
+      "I agree to the terms and conditions"
+    );
+    await userEvent.click(el);
```

Comment:
I prefer verbose names over shorthand for a few reasons:
1. Less things I have to parse as a dev.
2. We have designers and non-coders working on this library.

It's honestly difficult to know what all the shorthands mean in a codebase. It actually took me many years before I knew what a lot of them meant. I value verbosity for that reason alone. It doesn't cost anything and makes find and replace eaiser as well.

```suggestion
    const checkboxElement = await canvas.findByLabelText(
      "I agree to the terms and conditions"
    );
    await userEvent.click(checkboxElement);
```

---

We should also select this by role, not label text for this test because we're testing checkbox, not the internal `Field` component. For that reason, we want to do `findByRole('checkbox', { name: "I agree to the terms and conditions" })`.